lib/WTSI/NPG/HTS/TreePublisher.pm

```diff
@@ -60,6 +64,13 @@ has 'require_checksum_cache' =>
      documentation => 'A list of file suffixes for which MD5 cache files ' .
                       'must be provided and will not be created on the fly');
 
+has 'mlwh_json' =>
```

Comment:
This attribute should have a predicate "has_mlwh_json"

Reply:
Predicate "has_mlwh_json" added